custom/abdm/milestone_one/utils/response_util.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework.response import Response
+from rest_framework.status import (
+    HTTP_200_OK,
+    HTTP_400_BAD_REQUEST,
+    HTTP_500_INTERNAL_SERVER_ERROR,
+)
+
+success_response_keys = ["txnId", "healthIdNumber", "auth_methods", "token"]
+
+
+def parse_response(response_data):
+    if not response_data:
+        return Response({"error": "No valid response found"}, status=HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
Why are we returning 500 here? Is there a better HTTP code we can return to reflect that we got no response from the external system?

Reply:
This is added as a precautionary check. Ideally response is not expected to come empty.